boilerplates/vue/panda.config.ts

```diff
@@ -0,0 +1,25 @@
+import { defineConfig } from "@pandacss/dev";
+
+export default defineConfig({
+  // Whether to use css reset
+  preflight: true,
+
+  // Where to look for your css declarations
+  include: [
+    "./components/**/*.{js,jsx,ts,tsx,vue}",
+    "./layouts/**/*.{js,jsx,ts,tsx,vue}",
+    "./pages/**/*.{js,jsx,ts,tsx,vue}",
+    "./src/**/*.{js,jsx,ts,tsx,vue}",
+  ],
+
+  // Files to exclude
+  exclude: [],
+
+  // Useful for theme customization
+  theme: {
+    extend: {},
+  },
+
+  // The output directory for your css system
+  outdir: "files/styled-system",
+});
```

Comment:
Is this necessary? We already have `boilerplates/panda-css/files/panda.config.ts`